src/ocean/math/IEEE.d

```diff
@@ -338,7 +342,7 @@ public:
 }
 
 /// Return a snapshot of the current state of the floating-point status flags.
-IeeeFlags ieeeFlags() { return IeeeFlags.getIeeeFlags(); }
+IeeeFlags ieeeFlags() { return IeeeFlags(IeeeFlags.getIeeeFlags()); }
```

Comment:
& this line should also be possible to drop now that `naked` and `ret` are used ... ?